### PR TITLE
fix: 'empty screen' icons

### PR DESCRIPTION
**Pull request by deboer-tim**
### What does this PR do?

Fix each of the four empty screens to use the correct SVG icon and reduce code duplication via EmptyScreen.

Switched the order of the if statement in each of the lists because I thought this is more natural, and avoids passing array into each screen to confirm if it is empty.

The volumes empty screen does not appear before or after this change because filtered.length == 1 when there are no visible volumes, will continue investigating via separate PR.

### Screenshot/screencast of this PR

no-pods

### CODE REVIEW

File: packages/renderer/src/lib/pod/PodEmptyScreen.svelte [hunk 1]
```diff
@@ -1,33 +1,10 @@
...
-          ><Fa class="ml-3 h-5 w-5 cursor-pointer rounded-full text-3xl text-sky-800" icon="{faPaste}" /></button>
-      </div>
-    </div>
-  </div>
-</div>
+<EmptyScreen
+  icon="{PodIcon}"
+  title="No pods"
+  message="Run a first pod using the following command line:"
+  commandline="podman volume create myFirstVolume" />
```
Comment (benoitf) [inline packages/renderer/src/lib/pod/PodEmptyScreen.svelte]: looks like a copy/paste error

Comment (deboer-tim) [reply packages/renderer/src/lib/pod/PodEmptyScreen.svelte]: Fixed in new commit.

A little embarrassed at how many times I looked at that and didn't notice...

File: packages/renderer/src/lib/volume/VolumesList.svelte [hunk 2]
```diff
@@ -271,10 +271,10 @@ function computeInterval(): number {
     </table>
   </div>
   <div slot="empty" class="min-h-full">
-    {#if providerConnections.length > 0}
-      <VolumeEmptyScreen volumes="{$filtered}" />
-    {:else}
+    {#if providerConnections.length === 0}
       <NoContainerEngineEmptyScreen />
+    {:else if $filtered.length === 0}
```
Comment (benoitf) [inline packages/renderer/src/lib/volume/VolumesList.svelte]:
```suggestion
    {:else if $filtered.map(volumeInfo => volumeInfo.Volumes).flat().length === 0}
```

**Review by benoitf — APPROVED** [review]
if you can apply the correct if for volume lists

Tested, works fine
image
image
image

**Comment by benoitf** [comment]
looks like DCO check is reporting an error
```
Expected "Tim deBoer [deboer redhat.com]", but got "Tim deBoer [git tdeboer.ca]".
```

**Pull request merged**